app/views/users/show.html.erb

```diff
@@ -13,40 +14,28 @@
             </div>
 
             <div class="user-details">
-                <h3>User Name</h3>
-                <p>Number of post: x</p>
+                <h3><%=@user.name%></h3>
+                <p><%=@user.posts.count%></p>
             </div>
         </div>
 
         <div class="bio">
         <h3>Bio</h3>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer semper lacus eu nibh hendrerit, a rhoncus magna vulputate. Sed vel dolor porttitor felis aliquam tincidunt. Sed nibh orci, dictum sit amet sodales ac, pharetra in augue. Donec rhoncus vehicula magna. Sed laoreet fermentum tincidunt. Curabitur semper elit ac dui commodo, sed lobortis urna varius. Nulla risus neque, euismod ut ante ut, scelerisque luctus leo. Fusce congue dui at eros porta lacinia.
+            <%=@user.bio%>
         </div>
 
         <ul class="post-list">
+          <%@posts.each do |post|%>
```

Comment:
- [ ] As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models) requirement, kindly use the `recent_posts` method instead of showing all posts in the users details page.